source/manual/readmes.html.md

```diff
@@ -7,105 +7,73 @@ type: learn
 parent: "/manual.html"
 ---
 
-This is a guide to writing and maintaining README documents for GOV.UK's public repositories.
+This is a guide to writing and maintaining README documents for GOV.UK's public repositories. It's based on [guidance we have in the GDS Way for READMEs](https://gds-way.cloudapps.digital/manuals/readme-guidance.html#writing-readmes).
 
-## Who are READMEs for (aka what's the user need)?
+READMEs are for a technical audience. This could be a new starter or an existing developer. Using the following template will help to ensure each README is consistent, correct, short and actionable.
 
-- New starters to GDS
-- People joining a new team
-- People interested in our work
-- Developers on other teams
+Unless it helps someone to get started, any other content should be a separate file in the `docs/` directory for a repository. This also makes the content easier to discover in external search.
 
 ## Template for new READMEs
 
 ```markdown
 # App name
 
-One paragraph description and purpose.
+One or more paragraphs describing the app:
 
-## Screenshots (if there's a client-facing aspect of it)
+- What it's used for.
+- Any unusual dependencies it has.
 
```

Comment:
Suggest augmenting slightly with information from https://github.com/alphagov/readme-template/blob/main/README.md:

"# App name

- What the app does 
- Who the app is for
- Any unusual dependencies the app has
- Limitations of the app
- Links to other things as necessary

Reply:
Sure, we can add some of those in. I think it's worth agreeing what a valid example for each of them would be first, so we have a common understanding of what we mean by them.

> Links to other things as necessary

We generally put links in naturally where it's appropriate, throughout the document, so I'm not sure we need a specific prompt here. If we just mean bullet point links, we've got a "Further docs" section for that.

Do you think this would prompt someone to add something they otherwise wouldn't?

> Who the app is for

I think this is usually covered by "what the app does", which usually includes the phrase "frontend app" or "publishing app" or "API" (not really _for_ anyone). Frontend apps are for the general public, and publishing apps are for government publishers. This is common knowledge on GOV.UK, so I don't think we need to repeat it in each README.

Can you think of any exceptions?

> Any unusual prerequisites for the app to work

This sounds the same as the one above it - dependencies and prerequisites are synonyms to me. Most of the dependencies are automated away by our development environment, but maybe we could be clearer about what we mean by "unusual" what do you think?

> Limitations of the app

All apps are fundamentally limited 😉. What would be an example of this for you?

Reply:
- Re link to other things, I thought maybe links to other related apps or programs would be useful
- Re who the app is for, I cannot think of any exceptions right now, but it is best practice to specify who should use the thing you are documenting
- Re dependencies or prerequisites, i would say include anything that is not automated away by the development environment
- Re limitations, I don't know about an example. I guess it would something like "You can use this data analysis app with this type of data in this location, however it is not suitable for this other type of data"

Reply:
Sounds good to me.

Reply:
Great, I've added another commit with those changes.